lib/tasks/grant_superadmin.rake

```diff
@@ -0,0 +1,14 @@
+namespace :superadmin do
+  desc 'Grant the superadmin role to specified users'
+  task 'grant', [:user_list] => [:environment] do |_cmd, args|
+    raise ArgumentError, 'A list of users is required: `rake superadmin:grant[user1,user2,...]`' unless args.user_list
+    args.to_a.each do |u|
```

Comment:
What does `args` look like here, before you call `to_a` on it? Does rake already do some processing on the comma-separated list of usernames?